mmv1/third_party/terraform/services/container/resource_container_cluster.go.erb

```diff
@@ -2019,7 +2018,6 @@ func ResourceContainerCluster() *schema.Resource {
 						},
 						"cluster_dns_scope": {
 							Type:         schema.TypeString,
-							Default:      "DNS_SCOPE_UNSPECIFIED",
```

Comment:
Is removing this default really necessary to allow update in place? Generally an unspecified field in terraform should be equivalent to the zero value for the field, so leaving no default could change the behavior unexpectedly for users.

Reply:
`DNS_SCOPE_UNSPECIFIED` is indeed the `0` of that enum. It it part of our longer plan to simplify DNSConfig with terraform. But it is not actually related to this change. I reverted it back.